documentation/package.json

```diff
@@ -25,23 +25,25 @@
     "replace-2.x:clean-versioned": "rimraf versioned_sidebars/version-2.x-sidebars.json versioned_docs/version-2.x versions.json",
     "replace-2.x:copy-docs": "yarn apiref && yarn docusaurus docs:version 2.x && git checkout HEAD -- versions.json",
+    "replace-2.x:update-site-imports": "yarn replace \"'@site/docs/\" \"'@site/versioned_docs/version-2.x/\" versioned_docs/version-2.x/ --include '*.ts*,*.mdx' -r",
     "replace-2.x:update-sdk": "yarn replace \"\\'@cognite/sdk\\'\" \"''@cognite/sdk-2.x''\" versioned_docs/version-2.x/ --include '*.ts*' -r",
     "replace-2.x:update-reveal": "yarn replace \"\\'@cognite/reveal\\'\" \"''@cognite/reveal-2.x''\" versioned_docs/version-2.x/ --include '*.ts*' -r",
     "replace-2.x:update-runnable": "yarn replace '^```(js[x]?) runnable$' '```$1 runnable-2x' versioned_docs/version-2.x/ --include '*.mdx' -r",
-    "replace-2.x": "yarn replace-2.x:clean-versioned && yarn replace-2.x:copy-docs && yarn replace-2.x:update-sdk && yarn replace-2.x:update-reveal && yarn replace-2.x:update-runnable",
+    "replace-2.x": "yarn replace-2.x:clean-versioned && yarn replace-2.x:copy-docs && yarn replace-2.x:update-sdk && yarn replace-2.x:update-reveal && yarn replace-2.x:update-runnable && yarn replace-2.x:update-site-imports",
     "replace-3.x:clean-versioned": "rimraf versioned_sidebars/version-3.x-sidebars.json versioned_docs/version-3.x versions.json",
     "replace-3.x:copy-docs": "yarn apiref && yarn docusaurus docs:version 3.x && git checkout HEAD -- versions.json",
+    "replace-3.x:update-site-imports": "yarn replace \"'@site/docs/\" \"'@site/versioned_docs/version-3.x/\" versioned_docs/version-3.x/ --include '*.ts*,*.mdx' -r",
     "replace-3.x:update-sdk": "yarn replace \"\\'@cognite/sdk\\'\" \"''@cognite/sdk-3.x''\" versioned_docs/version-3.x/ --include '*.ts*' -r",
     "replace-3.x:update-reveal": "yarn replace \"\\'@cognite/reveal\\'\" \"''@cognite/reveal-3.x''\" versioned_docs/version-3.x/ --include '*.ts*' -r",
     "replace-3.x:update-runnable": "yarn replace '^```(js[x]?) runnable$' '```$1 runnable-3x' versioned_docs/version-3.x/ --include '*.mdx' -r",
-    "replace-3.x": "yarn replace-3.x:clean-versioned && yarn replace-3.x:copy-docs && yarn replace-3.x:update-sdk && yarn replace-3.x:update-reveal && yarn replace-3.x:update-runnable",
+    "replace-3.x": "yarn replace-3.x:clean-versioned && yarn replace-3.x:copy-docs && yarn replace-3.x:update-sdk && yarn replace-3.x:update-reveal && yarn replace-3.x:update-runnable && yarn replace-3.x:update-site-imports",
     "replace-latest-by-next": "yarn replace-3.x"
   },
   "dependencies": {
     "@cognite/reveal": "link:../viewer/dist",
     "@cognite/reveal-1.x": "npm:@cognite/reveal@1.x.x",
     "@cognite/reveal-2.x": "npm:@cognite/reveal@2.x.x",
-    "@cognite/reveal-3.x": "npm:@cognite/reveal@3.0.0-alpha.0",
+    "@cognite/reveal-3.x": "npm:@cognite/reveal@3.x.x",
```

Comment:
Does this pick up alpha versions too?

Reply:
I assume, yes. When building locally it asked to pick a version from the list. Not sure how it was resolved on documentation deployment but it is using proper version.

Reply:
Just checked and actually it's a wrong version format for alphas. Changed it to correct one. By the way, good site to check proper version tag: https://semver.npmjs.com/